src/puppetlabs/ezbake/dependency_utils.clj

```diff
@@ -116,6 +117,21 @@
         (assoc acc project-name (.getInputStream jar-file jar-entry)))
       acc)))
 
+(defn get-stream-from-project-jar
+  [lein-project file-path]
+  (let [project-jar-file (fs/file (:target-path lein-project)
```

Comment:
I can't think of any off the top of my head, but I also haven't dived deep into ezbake in almost a year.